src/libs/scene_utils/README.md

```diff
@@ -0,0 +1,88 @@
+# IoT TwinMaker Scene SDK
+
+## Use
+
+This package provides sample command line scripts that automate the creation of IoT TwinMaker scenes. Upload 3D assets and place them in a 3D scene connected to IoT data via augmented widgets (tag, motion indicator, etc.).
+
+## Install
+
+Make sure you are running the script in the `scene_utils` folder:
+
+```
+cd $GETTING_STARTED_DIR/src/libs/scene_utils
+npm install
+```
+
+# Samples:
+
+## 1. COOKIE FACTORY `samples/cookie_factory_sample/`
```

Comment:
discussed offline: might be missing some instructions here on having pre-created entities etc. in workspace

Reply:
Updated